<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="tomek-OptiPlex-790" ignored="0" name="Nopackage" tests="1" failures="1" timestamp="25 lip 2017 09:40:15 GMT" time="0.027" errors="0">
  <testcase name="f" time="0.011" classname="testngv2.testv1">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
	at testngv2.testv1.f(testv1.java:23)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- Nopackage -->
